### ISSUE TITLE
LT is having problems with the plural form of military

### ISSUE THREAD

**Issue by patchling** [languagetool-org/languagetool#955]
The sentence that is giving me problems is this one. I tried putting both spellings down one of them has a tooltip that might help with troubleshooting. So I have included a screenshot.

Some militarys are better at suppressing rebellions than others militaries are.

![languagetool style and grammar check](https://user-images.githubusercontent.com/16374144/37675465-8a1d00e2-2c32-11e8-9e69-63718222836e.png)

**Comment by Discostu36** [comment]
But in that case also "militaries" should be suggested.

**Comment by f-knorr** [comment]
I am not sure about that. The Levenshtein distance for 
* **military'** is 1
* **military s**  is 2
* **militaries** is 3
So, militaries is the least likely with respect to the other alternatives

**Comment by dpelle** [comment]
@f-knorr wrote:
> I am not sure about that. The Levenshtein distance for [...]
But I don't think that this bug bug is about Levenshtein here.  The popop window in the screenshot says:
```
Did you mean "military", the plural form of the noun "military"?
```
... which is clearly non-sense.  Why does it say "the plural form of" when it gives a singular noun?
For other kinds of typos, the message does not say "the plural form of".

**Comment by Discostu36** [comment]
@fknorr: Yes, but this is not a typo but a grammar mistake.

**Comment by f-knorr** [comment]
Well, i have to admit that my initial assumption w.r.t. the Levenshtein distance was wrong. 
The standalone application shows also the expected "militaries"
![grafik](https://user-images.githubusercontent.com/16450958/37737196-62b3988c-2d53-11e8-8fd8-96ac3b60a37e.png)
I think the classification as a spelling error is, indeed, correct, as there is no "militarys" in the dictionary.

@dpelle : I admit the error message is somewhat confusing, but actually "Did you mean "military", the plural form of the noun "military"?" is correct because military is both the singular and plural form.

To sum it up, I do not see any bug here.

**Comment by f-knorr** [comment]
Closing this issue as I cannot see an error here. If you disagree, please reopen and describe the desired  expected behavior.
